### ISSUE TITLE
kernel-update Fedora 41

### ISSUE THREAD

**Issue by ssergey2018** [gregory-lee-bartholomew/fedora-on-zfs#4]
dnf5 has a different sitnax from the usual dnf. When trying to run kernel-update on the updated Fedora 41 kernel-update I saw the following message
Unknown argument "--upgrades" for command "dnf5". Add "--help" for more information about the arguments.
The argument is available for commands: changelog, info, list, repoquery. (It has to be placed after the command.)
No kernel updates found, aborting ...

**Comment by gregory-lee-bartholomew** [comment]
Yes. Sorry about that. I noticed the error and updated that script in this repo a couple of days ago. You can find it here: https://github.com/gregory-lee-bartholomew/fedora-on-zfs/tree/main/supplements

You should be able to download just that `kernel-update` script and copy it over the older version that is under `/usr/local/bin`. (You might as well copy all three scripts -- `dnf`, `zfs-update`, and `kernel-update` -- to be sure you have the latest versions of them all.)

Thanks for reporting this issue. I'll leave it open in case others encounter the same problem.

**Comment by gregory-lee-bartholomew** [comment]
BTW, if you (or others) need to bypass that `dnf` filter temporarily, the way to do so is by using `command -p dnf [arg ...]`. Or, to permanently remove the filter, you can remove those scripts from `/usr/local/bin`. I'll add a hint about how to bypass the filter to that script's output.

**Comment by ssergey2018** [comment]
Thanks for the quick reply. I didn't know that there was an updated version with a fix for this error.
good luck!!!
